lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -447,6 +449,22 @@ class GatewayProvider(
             )
         }
 
+        @JvmStatic
+        fun makeTestnetClient(testnetId: StarknetChainId): GatewayProvider =
```

Comment:
Should we have some kind of unit tests for these constructors?

Reply:
Up until now we didn't, not sure what we could do except maybe just create a client and check its chainId (+ I'm really not sure how to test custom httpService, if at all)

Reply:
Perhaps let's create an issue for it then and we can cover it sometime later.

Reply:
https://github.com/software-mansion/starknet-jvm/issues/206